main.tf

```diff
@@ -46,7 +47,7 @@ resource "aws_route53_record" "main" {
 
 resource "aws_acm_certificate_validation" "main" {
   certificate_arn         = "${aws_acm_certificate.main.arn}"
-  validation_record_fqdns = aws_route53_record.main.*.fqdn
+  validation_record_fqdns = ["${aws_route53_record.main.*.fqdn}"]
```

Comment:
This looks like 0.11 syntax. This module is now on 0.12 so the above change would break on 0.12.